neo/app/db/Migration_20160814.scala

```diff
@@ -27,10 +27,6 @@ private[db] class Migration_20160814() {
          |}
          |
          |create dataset $berryMeta(berry.metaType) if not exists primary key name;
-         |
-         |upsert into ${if (conn.isInstanceOf[AsterixAQLConn]) "dataset" else ""} $berryMeta (
-         |  ${Json.toJson(DataSetInfo.write(twitterInfo))}
```

Comment:
Furthermore, could you also try to remove the `TwitterDataStore`? It should be only used by test cases, then let's just move it to test only.